.eslintrc.json

```diff
@@ -17,6 +17,7 @@
     "no-console": "warn",
     "func-names": "off",
     "no-process-exit": "off",
-    "class-methods-use-this": "off"
+    "class-methods-use-this": "off",
+    "arrow-body-style": "warn"
```

Comment:
What was the reason for adding this rule?

Reply:
Eslint was giving an error for arrow function inside describe and tests were not taking place, so I added, but it was when I was trying to use mockingoose I guess, I don't remember exactly. 
Now I tried to run without this rule and it is working fine, so I removed it.